developer-extension/src/panel/hooks/useEvents/eventFilters.ts

```diff
@@ -45,17 +46,30 @@ export function applyEventFilters(filters: EventFilters, events: SdkEvent[], fac
 
 function filterExcludedFacets(
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I would add a unit test.